### ISSUE TITLE
Authentication against Active Directory LDAP server crashes verdaccio

### ISSUE THREAD

**Issue by GlennMatthys** [Alexandre-io/verdaccio-ldap#31]
Authentication succeeds but verdaccio crashes afterwards. Looks like the "groups" passed back to verdaccio isn't an array:

```
May 25 10:20:49 websrv1 verdaccio: http <-- 200, user: glenn.matthys(*redacted* via ::1), req: 'POST /-/verdaccio/login', bytes: 66/1256
May 25 10:20:49 websrv1 verdaccio: fatal--- uncaught exception, please report this
May 25 10:20:49 websrv1 verdaccio: TypeError: (groups || []).concat is not a function
May 25 10:20:49 websrv1 verdaccio: at authenticatedUser (/usr/lib/node_modules/verdaccio/src/lib/auth.js:438:32)
May 25 10:20:49 websrv1 verdaccio: at /usr/lib/node_modules/verdaccio/src/lib/auth.js:117:26
May 25 10:20:49 websrv1 verdaccio: at sendResult (/usr/lib/node_modules/verdaccio-ldap/node_modules/ldapjs/lib/client/client.js:1395:12)
May 25 10:20:49 websrv1 verdaccio: at messageCallback (/usr/lib/node_modules/verdaccio-ldap/node_modules/ldapjs/lib/client/client.js:1421:16)
May 25 10:20:49 websrv1 verdaccio: at /usr/lib/node_modules/verdaccio-ldap/node_modules/ldapjs/lib/client/client.js:1282:14
May 25 10:20:49 websrv1 verdaccio: at Array.forEach (native)
May 25 10:20:49 websrv1 verdaccio: at Client._onClose (/usr/lib/node_modules/verdaccio-ldap/node_modules/ldapjs/lib/client/client.js:1272:19)
May 25 10:20:49 websrv1 verdaccio: at Socket.g (events.js:292:16)
May 25 10:20:49 websrv1 verdaccio: at emitOne (events.js:96:13)
May 25 10:20:49 websrv1 verdaccio: at Socket.emit (events.js:188:7)
May 25 10:20:49 websrv1 verdaccio: at TCP._handle.close [as _onclose] (net.js:509:12)
```

config.yaml:
```
auth:
  ldap:
    type: ldap
    client_options:
      adminDn: "*redacted*"
      adminPassword: "*redacted*"
      url: "ldap://*redacted*.local"
      searchBase: "DC=*redacted*,DC=*redacted*"
      searchFilter: "(sAMAccountName={{username}})"
      groupDnProperty: "distinguishedName"
      groupSearchBase: "DC=*redacted*,DC=*redacted*"
      searchAttributes: ['*', 'memberOf']

```

**Comment by GlennMatthys** [comment]
[Looks like the authentication callback is called with the UnbindResponse from LdapClient.close](https://github.com/Alexandre-io/verdaccio-ldap/blob/master/index.js#L64).

**Comment by GlennMatthys** [comment]
Possible fix in PR #32

**Comment by GlennMatthys** [comment]
PR #32 merged.
